src/AspNet.Security.OAuth.Kloudless/KloudlessAuthenticationConstants.cs

```diff
@@ -30,5 +30,17 @@ public static class Claims
             public const string ProxyConnection = "urn:kloudless:proxy_connection";
             public const string Active = "urn:kloudless:active";
         }
+
+        public static class Scopes
+        {
+            public const string Any = "any";
+            public const string Storage = "storage";
+            public const string Calendar = "calendar";
+            public const string Email = "email";
+            public const string CRM = "crm";
```

Comment:
```suggestion
            public const string Crm = "crm";
```

Reply:
This is an acronym, that's why I'm using all uppercase here

Reply:
.NET convention is to use first-letter capital and the rest lowercase ([docs](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1709?view=vs-2019)) for acronyms of 3 letters in length or more.

> By convention, two-letter acronyms use all uppercase letters, and acronyms of three or more characters use Pascal casing. The following examples use this naming convention: 'DB', 'CR', 'Cpa', and 'Ecma'. The following examples violate the convention: 'Io', 'XML', and 'DoD', and for non-parameter names, 'xp' and 'cpl'.